pokemon-checker/src/App.tsx

```diff
@@ -4,20 +4,20 @@ import "./components/PokemonSearch/PokemonSearchResults";
 import PokemonSearchResults from "./components/PokemonSearch/PokemonSearchResults";
 import SearchBox from "./components/SearchBar/SearchBar";
 import PokemonDisplay from "./components/PokemonDisplay/PokemonDisplay";
-import PokemonSearchObj from "./components/PokemonSearch/PokemonSearchResults";
-import { getAllPokemon } from "./services/PokemonService";
+import { PokemonService } from "./services/PokemonService";
+import PokemonDTO from "./DataTransferObjects/PokemonDTO";
 
 type AppState = {
-  pokemonList: typeof PokemonSearchObj[];
+  pokemonList: PokemonDTO[];
```

Comment:
There were some messy type errors here when we used `typeof PokemonSearchObj[]`, so I've decided that our state should be bound to data and not functions that return `JSX.Element`.